src/design/demo/image-deck.twig

```diff
@@ -0,0 +1,12 @@
+{% embed '@cloudfour/objects/deck/deck.twig' with {
+  class: 'o-deck--2-column@l'
+} %}
+  {% block content %}
+    {% for image in images %}
+      <figure>
+        <img src="{{image.src}}" alt="">
```

Comment:
Should we include alt text for these images? We'll want to when we use them on the Wordpress site. Should the pattern library define and expose those alt descriptions?

Reply:
Ahh I see we're just pulling the name from the filename. I was imagining there was a JSON file somewhere of data. Hmm, what do you think? Is there a good place to define those alt descriptions in the patterns?

Reply:
The WordPress site will have a lot more information about the article the image is supposed to represent, so I'm not sure that specifying `alt` text here will be very useful. I think it may actually complicate implementation.

Reply:
Cool, that works for me :+1: